RoddyCore/test/src/de/dkfz/roddy/config/ConfigurationValueSpec.groovy

```diff
@@ -24,4 +44,32 @@ class ConfigurationValueSpec extends Specification {
         new ConfigurationValue("b", "abc", "string") | null        | evString
         new ConfigurationValue("c", 1)               | null        | evInt
     }
+
+    def "test variable replacement for toFile()"() {
+        when:
+        ExecutionContext context = contextResource.createSimpleContext(ConfigurationValueSpec.class)
+        def values = context.configurationValues
+        values << new ConfigurationValue(context.configuration, "b", 'abc')
+        values << new ConfigurationValue(context.configuration, "c", 'def')
+        values << new ConfigurationValue(context.configuration, "a", '/tmp/${projectName}/${b}/something${c}fjfj/${dataSet}')
+        values << new ConfigurationValue(context.configuration, "complex", '/$USERHOME/$USERNAME/$USERGROUP')
```

Comment:
Also, what about incorrect variable replacement in
```groovy
'$USERNAMEbutDontReplaceThisUsernameButReportAsPlainDollarUsageWarning
```

Reply:
Configuration value issues tests are covered at another place!